ui/scss/temp.scss

```diff
@@ -92,7 +92,7 @@ body, var {
   top: 0;
   left: 0;
   width: 100%;
-  z-index: 100;
+  z-index: 99;
```

Comment:
Please make sure the session times out, and we gray out the screen, everything is grayed out except the session timeout dialog.

Reply:
Yes, actually this change fixes it. Also should have been a separate commit.